src/app/api/send-email/route.ts

```diff
@@ -19,7 +19,7 @@ export async function POST(req) {
     // Send the email using sg
     await sg.send({
       to: to,
-      from: "taaseenmkhan@gmail.com", // Verified sender email in sg
```

Comment:
probably wanna store this as an env variable so we can hotswap the email, but this is fine as it is for  our purposes